shop/myFlywayConfig.conf

```diff
@@ -0,0 +1,5 @@
+flyway.user=sa
```

Comment:
You don't need this flyway configuration. You can put those properties in **application.properties** and then the app will automatically run your migrations when you start it. If you don't know the properties names ask Adela, if I remember right she already did this

Reply:
I removed it and moved all the flyway config to application.properties file.